src/Uno.UWP/ApplicationModel/Appointments/AppointmentStore.Android.cs

```diff
@@ -0,0 +1,179 @@
+﻿#if __ANDROID__

```

Comment:
```suggestion
#nullable enable

﻿#if __ANDROID__

```

Reply:
It cannot be nullable, as RequestStoreAsync returns AppointmentStore, not AppointmentStore?, and can return null.